catalog/spatial/geocoding/spatial-geocoding-index/src/main/java/org/codice/ddf/spatial/geocoding/index/GeoNamesQueryLuceneDirectoryIndex.java

```diff
@@ -12,7 +12,7 @@
  * <http://www.gnu.org/licenses/lgpl.html>.
  **/
 
-package org.codice.ddf.spatial.geocoding.query;
+package org.codice.ddf.spatial.geocoding.index;
```

Comment:
You could still leave these in the same package, just move them to the new project.